script/cluster-kind.mk

```diff
@@ -15,36 +15,36 @@ ${CLUSTER_CONFIG}:
 		--name ${CLUSTER_NAME} \
 		--config=./docs/user/manifests/kubeapps-local-dev-apiserver-config.yaml \
 		--retain \
-		--wait 10s
-	kubectl apply --kubeconfig=${CLUSTER_CONFIG} -f ./docs/user/manifests/kubeapps-local-dev-users-rbac.yaml
-	kubectl apply --kubeconfig=${CLUSTER_CONFIG} -f ./docs/user/manifests/ingress-nginx-kind-with-large-proxy-buffers.yaml
+		--wait 10s || true
+	kubectl apply --kubeconfig=${CLUSTER_CONFIG} -f ./docs/user/manifests/kubeapps-local-dev-users-rbac.yaml || true
+	kubectl apply --kubeconfig=${CLUSTER_CONFIG} -f ./docs/user/manifests/ingress-nginx-kind-with-large-proxy-buffers.yaml || true
 	# TODO: need to add wait for condition=exists or similar - https://github.com/kubernetes/kubernetes/issues/83242
 	sleep 5
 	kubectl wait --kubeconfig=${CLUSTER_CONFIG} --namespace ingress-nginx \
 		--for=condition=ready pod \
 		--selector=app.kubernetes.io/component=controller \
-		--timeout=120s
+		--timeout=120s || true
 
 cluster-kind: ${CLUSTER_CONFIG}
 
 # dex will be running on the same node as the API server in the dev environment, so we can
 # reuse the key and cert from the apiserver, which already includes v3 extensions
 # for the correct alternative name (using the IP address).
 devel/dex.crt:
-	kubectl -n kube-system cp kube-apiserver-kubeapps-control-plane:etc/kubernetes/pki/apiserver.crt ./devel/dex.crt
+	docker cp kubeapps-control-plane:/etc/kubernetes/pki/apiserver.crt ./devel/dex.crt || true
```

Comment:
I wasn't able to perform a `kubectl cp`... it started complaining about "executable file not found in $PATH”: unknown", so I just decided to go through Docker to perform this operation

Reply:
Ahh, now everything makes sense. I wasn't aware of the k8s move to distroless. Thank you for letting me know :)